tests/test_cli.py

```diff
@@ -302,13 +302,9 @@ def test_max_nodes_unsat(self):
             "--max-nodes",
             "30",
         ]
-        err_pattern = r"""fandango:ERROR: Population did not converge to a perfect population
-fandango:ERROR: Only found (\d) perfect solutions, instead of the required 10"""
-        out_pattern = (
-            r"""(aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa\.\d+\n)*"""
-        )
+        err_pattern = r"fandango:ERROR: Only found (0) perfect solutions, instead of the required 10"
         out, err, code = run_command(command)
-        self.assertRegex(out, out_pattern, f"out: {out}")
+        self.assertEqual(out, '', f"out: {out}")
         self.assertRegex(err, err_pattern, f"err: {err}")
```

Comment:
We don't need regex here anymore, I don't think:
```suggestion
        out, err, code = run_command(command)
        self.assertEqual(out, '', f"out: {out}")
        self.assertEqual(err, "fandango:ERROR: Only found 0 perfect solutions, instead of the required 10")
```